blackbird/utils/configread.py

```diff
@@ -661,7 +663,8 @@ def is_log(value):
 
            Recommended giving he full path including the file name.
 
-    :param str value: log file path. You can specify not only absolute path but also relational path.
+    :param str value: log file path. You can specify not only absolute path
+           but also relational path.
```

Comment:
Sphinx style indent is four. You man write this param description below.

1. 
```
:param str value: log file path. You can specify not only absolute path
    but also relational path.
```

2. 
```
:param str value:
    log file path. You can specify not only absolute path
    but also relational path.
```

I'd like to style2.